test/backend/DataVaultTestSuite.scala

```diff
@@ -45,14 +46,35 @@ class DataVaultTestSuite extends PlaySpec {
       }
 
       "with Google Cloud Storage Vault" should {
+        val uri = new URI("gs://neuroglancer-fafb-data/fafb_v14/fafb_v14_orig")
+        val vaultPath = new VaultPath(uri, GoogleCloudDataVault.create(RemoteSourceDescriptor(uri, None)))
         "return correct response" in {
-          val uri = new URI("gs://neuroglancer-fafb-data/fafb_v14/fafb_v14_orig")
-          val vaultPath = new VaultPath(uri, GoogleCloudDataVault.create(RemoteSourceDescriptor(uri, None)))
+
           val bytes = (vaultPath / dataKey).readBytes(Some(range))(globalExecutionContext).get(openFoxJustification)
 
           assert(bytes.length == range.length)
           assert(bytes.take(10).sameElements(Array(-1, -40, -1, -32, 0, 16, 74, 70, 73, 70)))
         }
+
+        "return empty box" when {
+          "requesting a nox-existent object" in {
+            val result = (vaultPath / "non-existent-key").readBytes()(globalExecutionContext).await()
+            assert(result.isEmpty)
```

Comment:
Failure boxes also have `isEmpty == true`. I think this needs a different kind of check (e.g. use `match`)